portal-impl/src/com/liferay/portal/spring/hibernate/PortalHibernateConfiguration.java

```diff
@@ -264,6 +268,14 @@ protected Configuration newConfiguration() {
 					new FlushEventListener[] {
 						NestableFlushEventListener.INSTANCE
 					});
+				eventListeners.setLoadEventListeners(
```

Comment:
@shuyangzhou I would like to highlight that we need two different load listeners here:
PostLoadListener is invoked after loading from database
LoadListener is invoked after loading from session, we need this to protect the object retrieved from first level cache of hibernate directly.

Tina.